PVList.md

```diff
@@ -0,0 +1,364 @@
+    TempPCB{-Mon|Prs-Cte} <br>
+    TempPCBMax{-Mon|Prs-Cte} <br>
+    TempDCDC{-Mon|Prs-Cte} <br>
+    TempDCDCMax{-Mon|Prs-Cte} <br>
+    TempComExCPU{-Mon|Prs-Cte} <br>
+    TempComExPCH{-Mon|Prs-Cte} <br>
+    TempComExDDR{-Mon|Prs-Cte} <br>
+    Volt1V0{-Mon|Prs-Cte} <br>
+    Volt1V8{-Mon|Prs-Cte} <br>
+    Volt3V3{-Mon|Prs-Cte} <br>
+    Volt5V{-Mon|Prs-Cte} <br>
+    VoltVBAT{-Mon|Prs-Cte} <br>
+    Volt12VInt{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Status-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+    SoftRst-Cmd <br>
+    SoftRstSts-Mon <br>
+<br>
+</details>
+
+## Cooling Unit
+
+### `$(AREA):$(DIS)-CU{Top|Bot}:`
+<details>
+<summary>sensors</summary>
+   TempLM75{-Mon|Prs-Cte} <br>
+   TempLM75_2{-Mon|Prs-Cte} <br>
+   Volt3V3{-Mon|Prs-Cte} <br>
+   Volt12V{-Mon|Prs-Cte} <br>
+   Volt12V_1{-Mon|Prs-Cte} <br>
+   SFan1{-Mon|Prs-Cte} <br>
+   SFan2{-Mon|Prs-Cte} <br>
+   SFan3{-Mon|Prs-Cte} <br>
+   SFan4{-Mon|Prs-Cte} <br>
+   SFan5{-Mon|Prs-Cte} <br>
+   SFan6{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Status-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+    Pwr-Mon <br>
+    SoftRst-Cmd <br>
+    SoftRstSts-Mon <br>
+<br>
+</details>
+
+## Power Module
+### `$(AREA):$(DIS)-PM-{2|4}:`
+<details>
+<summary>sensors</summary>
+    TempDCDC{-Mon|Prs-Cte} <br>
+    TempPath{-Mon|Prs-Cte} <br>
+    TempCooler{-Mon|Prs-Cte} <br>
+    TempTrafo{-Mon|Prs-Cte} <br>
+    VoltMP{-Mon|Prs-Cte} <br>
+    VoltSMP{-Mon|Prs-Cte} <br>
+    VoltSMPP{-Mon|Prs-Cte} <br>
+    VoltPP{-Mon|Prs-Cte} <br>
+    Volt12VHHV{-Mon|Prs-Cte} <br>
+    CurrSum{-Mon|Prs-Cte} <br>
+    CurrCh01{-Mon|Prs-Cte} <br>
+    CurrCh02{-Mon|Prs-Cte} <br>
+    CurrCh03{-Mon|Prs-Cte} <br>
+    CurrCh04{-Mon|Prs-Cte} <br>
+    CurrCh05{-Mon|Prs-Cte} <br>
+    CurrCh06{-Mon|Prs-Cte} <br>
+    CurrCh07{-Mon|Prs-Cte} <br>
+    CurrCh08{-Mon|Prs-Cte} <br>
+    CurrCh09{-Mon|Prs-Cte} <br>
+    CurrCh10{-Mon|Prs-Cte} <br>
+    CurrCh11{-Mon|Prs-Cte} <br>
+    CurrCh12{-Mon|Prs-Cte} <br>
+    CurrCh13{-Mon|Prs-Cte} <br>
+    CurrCh14{-Mon|Prs-Cte} <br>
+    CurrCh15{-Mon|Prs-Cte} <br>
+    CurrCh16{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Status-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+    SoftRst-Cmd <br>
+    SoftRstSts-Mon <br>
+<br>
+</details>
+
+## AMC
+### `$(AREA):$(DIS)-AMC-{1,3..12}:`
```

Comment:
Add a small line of text here after the header explaining these are AFCv3.1. And about the AFCv4 below.